tasks/managed/collect-data/tests/test-collect-data-print-pipeline-ref-nongit.yaml

```diff
@@ -143,10 +231,11 @@ spec:
               #!/usr/bin/env bash
               set -eux
 
-              touch "$(workspaces.data.path)/mock_curl.txt"
```

Comment:
this is a weird test.. touch a file then see if it is empty. fail if it isn't :thinking: 
anyways, not introduced in this change, so can resolve without action